frontend/src/pages/notebookController/screens/admin/StopAllServersButton.tsx

```diff
@@ -42,14 +49,14 @@ const StopAllServersButton: React.FC<StopAllServersButtonProps> = ({ users }) =>
       >
         Stop all workbenches ({serverCount})
       </Button>
-      {showModal && (
+      {showModal && loaded ? (
```

Comment:
```suggestion
      {showModal && loaded && (
```
we don't need to use ternary here